banjo/conf/schema.xml

```diff
@@ -219,7 +219,10 @@
    <field name="additional_contributor" type="text_general" indexed="true" stored="true"/>
    <field name="additional_creator" type="text_general" indexed="true" stored="true"/>
    <field name="access_agreement" type="string" indexed="true" stored="true"/>
-
+   <field name="access_agreement_reason_for_change" type="string" indexed="true" stored="true"/>
+   <field name="access_condition_reason_for_change" type="string" indexed="true" stored="true"/>
+   <field name="commercial_status_reason_for_change" type="string" indexed="true" stored="true"/>
+   <field name="access_comments" type="string" indexed="true" stored="true"/>

```

Comment:
text_general is tokenised (ie: the text being processed is broken on space boundaries and added to the index individual words). So you can find records by searching using individual words and phrases. 

string is indexed as is and can only be searched as an exact match on the original text.